src/librustdoc/html/static/js/main.js

```diff
@@ -699,11 +695,6 @@ function preLoadCss(cssUrl) {
             }
 
         });
-
-        const pageId = getPageId();
```

Comment:
I'm a bit surprised that even with this removed, when we arrive on a page targetting a hidden element, it will work. Do we have a GUI test to enforce it already?

Reply:
Good catch. Thanks for the explanations!